velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "HdfsWriteFile.h"
+#include <hdfs/OutputStream.h>
+#include <hdfs/hdfs.h>
+#include <iostream>
+#include <regex>
+
+namespace facebook::velox {
+
+HdfsWriteFile::HdfsWriteFile(std::string& path) {
+  struct hdfsBuilder* builder = hdfsNewBuilder();
+  // read hdfs client conf from hdfs-client.xml from LIBHDFS3_CONF
```

Comment:
Comments must start with a capital letter and end with a period. Please, fix throughout the PR.